src/index_html/index.html

```diff
@@ -11,6 +11,19 @@
   <body>
       <Nav>
         <img src="https://cdn.evbuc.com/eventlogos/188084824/acastandardcirclefullname.png" class="logo" alt="austin coding academy logo">
+<<<<<<< HEAD:src/index_html/index.html
```

Comment:
Overall, things look pretty good. You want to make sure you want to get rid of lines that start with these arrows `<` and `>` otherwise they will be part your html